src/__init__.py

```diff
@@ -1,3 +1,3 @@
-from src.commands.install import installation_exceptions
-from .commands import install
+from src.commands import *
```

Comment:
i dislike the asterisk as well, since it may pollute the namespace when code is changed inside the module with it